system-variables.md

```diff
@@ -1330,7 +1330,9 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 > **Note:**
 >
```

Comment:
Can we swap the sequence of the two notes to let users understand what `temp-dir` is and then tell them why the configuration is recommended before the upgrade?